client/my-sites/customer-home/cards/features/go-mobile/index.jsx

```diff
@@ -29,9 +29,9 @@ export const GoMobile = ( { email, sendMobileLoginEmail } ) => {
 		<Card className="go-mobile customer-home__card">
 			<div className={ classnames( 'go-mobile__row', { 'has-2-cols': showOnlyOneBadge } ) }>
 				<div className="go-mobile__title">
-					<CardHeading tagName="h2">{ translate( 'WordPress app' ) }</CardHeading>
+					<CardHeading tagName="h2">{ translate( 'Get the WordPress app' ) }</CardHeading>
 					<h3 className="go-mobile__subheader customer-home__card-subheader">
-						{ translate( 'Make updates on the go.' ) }
+						{ translate( 'Inspiration strikes any time, anywhere.' ) }
```

Comment:
@arthur791004  I'd suggest this should be "Inspiration strikes anytime, anywhere."   e.g. `anytime` should be one word to be consistent with `anywhere`. :) 

Using any time as two words is correct in English, but it flows better to have consistency of anytime, anywhere.  https://www.grammarly.com/blog/anytime-any-time.

Reply:
Okay, I just follow the original issue. I'll change to `anytime`.